src/Exercise02AI.ipynb

```diff
@@ -179,23 +179,20 @@
     "\n",
     "Die rekursive `minimax`-Funktion betrachtet bei einem gegebenen Board alle möglichen Stellungen (Boards) nach einer gegebenen Anzahl von Halbzügen (Tiefe) und gibt die beste Evaluierung (`evaluate`-Funktion) für den aktuellen Spieler zurück. Hierfür bekommt die Funktion als Argumente das aktuelle Board (`board`), die Evaluierungstiefe in Halbzügen (`depth`) und die Evaluierung des übergebenen Boards (`current_evaluation`). Zurückgegeben wird ein Tupel bestehend aus der besten Evaluierung und des Halbzuges, der auf den Pfad zu dieser Evaluierung führt.\n",
     "\n",
-    "Die Konstante `LIMIT` wird als obere (+) beziehungsweise untere (-) Schranke für den Vergleich innerhalb der ersten Iteration verwendet.\n",
+    "Die Konstante `LIMIT` wird als obere (`+`) beziehungsweise untere (`-`) Schranke für den Vergleich innerhalb der ersten Iteration verwendet.\n",
     "\n",
-    "Folgende Fälle werden unterschieden:\n",
+    "Weiter existiert die Funktion `early_abort_evaluation`, welche dieselben Argument annimmt, wie die zuvor beschriebene. Diese wird zu Beginn der `minimax`-Funktion aufgerufen und bricht unter der Voraussetzungen, dass das Spiel beendet ist mithilfe der folgenden Fallunterscheidung den Funktionsaufruf und damit die Rekursion ab:\n",
```

Comment:
Würde das noch umstrukturieren und in zwei Abschnitte aufteilen, finde es sonst etwas verwirrend :-)

Reply:
Fixed in e425aae.